pkg/flare/archive.go

```diff
@@ -371,6 +376,23 @@ func zipExpVar(tempDir, hostname string) error {
 	return err
 }
 
+func zipSystemProbeStats(tempDir, hostname string) error {
```

Comment:
Since we're already adding the system probe stats to the status here: 
https://github.com/DataDog/datadog-agent/pull/5339/files#diff-a5666326aefdf43a8e89fd4871449fa0L87 

and then zipping the entire status into a file here: 
https://github.com/DataDog/datadog-agent/blob/cd3678473cdc4bd88167380e40e14cf70440e3fc/pkg/flare/archive.go#L259 
(using https://github.com/DataDog/datadog-agent/blob/cd3678473cdc4bd88167380e40e14cf70440e3fc/pkg/status/status.go#L93).

Wouldn't the stats already be available in the flare? Isn't this redundant? I might be missing something.

Reply:
@truthbk The reasoning behind this is that we will pull ALL stats from expvar for the system-probe. 

In the status function we end up only printing out the number of client connections, and due to the number of the returned variables we did not want to put in everything in the status page to make it too large/messy. 

Does that make sense?

Reply:
Yeah, I see the template doesn’t provide full context. So my next question is, wouldn't it make sense to dump the entire raw stats map? I mean just like this is happening for system probe stuff, we might also be missing some details from other components as well.